src/ui/mainwindow.cc

```diff
@@ -1280,16 +1302,25 @@ void MainWindow::applyQtStyleSheet( QString const & addonStyle, QString const &
   }
 #endif
 
+#if !defined( Q_OS_WIN )
+  if ( interfaceStyle == "Default" ) {
+  #if defined( Q_OS_LINUX )
+    QApplication::setStyle( QStyleFactory::create( defaultInterfaceStyle ) );
+  #else
+    QApplication::setStyle( QStyleFactory::create( "Fusion" ) );
+  #endif
+  }
+  else {
+    if ( QStyleFactory::keys().contains( interfaceStyle ) ) {
+      QApplication::setStyle( QStyleFactory::create( interfaceStyle ) );
+    }
+  }
+#endif
+
   QFile builtInCssFile( ":qt-style.css" );
   builtInCssFile.open( QFile::ReadOnly );
   QByteArray css = builtInCssFile.readAll();
 
-#if defined( Q_OS_MAC )
-  QFile macCssFile( ":qt-style-macos.css" );
-  macCssFile.open( QFile::ReadOnly );
-  css += macCssFile.readAll();
-#endif
-
```

Comment:
removed?

Reply:
Yes, the `qt-style-macos.css` is empty, and I think we would use never it.